services/movie_session.py

```diff
@@ -42,3 +41,14 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
The `delete_movie_session_by_id` function uses `get()`, which can raise a `DoesNotExist` exception if no movie session is found with the given ID. Consider handling this exception to avoid potential runtime errors.